### PR TITLE
Link to "What Does My Site Cost" and "Webpagetest" in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by toolness**
Recently found out about these great tools and thought it might be useful to mention them in our `CONTRIBUTING.md` file!

### CODE REVIEW

**Comment by mmmavis** [comment]
Travis gave a sad face from timeout error.  I'm restarting the build again.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -126,6 +128,8 @@ changes haven't introduced any [Mixed Content][] warnings.
 [js]: https://github.com/MozillaFoundation/javascript-style-guide
 [synonym]: https://help.github.com/articles/closing-issues-via-commit-messages
 [PageSpeed Insights]: https://developers.google.com/speed/pagespeed/insights/
+[WebPagetest]: http://www.webpagetest.org/
+[What Does My Site Cost]: http://whatdoesmysitecost.com/
 [Tenon]: http://tenon.io/
 [ngrok]: https://ngrok.com/
 [Mixed Content]: https://developer.mozilla.org/en-US/docs/Security/MixedContent
```
Comment (mmmavis) [inline CONTRIBUTING.md]: interesting, I didn't know we can list links this way :+1:

**Comment by mmmavis** [comment]
Yay R+ thanks @toolness

File: CONTRIBUTING.md (hunk 1 above)
Comment (toolness) [inline CONTRIBUTING.md]: Yeah! I actually prefer this way when the .md file is also going to be viewed as plaintext, which is often the case with READMEs, etc.

**Comment by toolness** [comment]
Thanks mavis!

> Travis gave a sad face from timeout error. I'm restarting the build again.

This happened today when I merged your #1072 too, so I increased the timeout for the offending test in c4742584eb36297fdc05f74f6cc5a3a53eee812e (the test is taking 570ms on my desktop, so I could see how it might take more than 2s on some travis runs).

**Pull request merged**